Automattic/jetpack
# Sync: Improve the taxonomies blacklist by adding more conflicting ones

This PR broadens the list of blacklisted taxonomies. This list was compiled by:

* Getting a full list of all the actions and filters that contain the taxonomy slug in the filter/action name.
* Going through all the filters/actions that might conflict any of the above by specifying a particular taxonomy slug.
* Making the list items unique and sorting them.

#### Changes proposed in this Pull Request:
* Sync: Improve the taxonomies blacklist by adding more conflicting ones

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Related with p5TWut-hM-p2 and p7rcWF-12N-p2, see #12957 for initial implementation.

#### Testing instructions:
* **Important:** Make sure we're not blacklisting any important core taxonomy.
* Checkout this branch.
* Register a forbidden custom taxonomy (for example bloginfo_rss).
* Create a term in that taxonomy, and verify it doesn't incrementally sync.
* Trigger full sync, and verify it doesn't sync forbidden taxonomy terms.
* Make sure tests pass.

#### Proposed changelog entry for your changes:
* Sync: Improve the taxonomies blacklist by adding more conflicting ones
